tests/installation/launch_installation.pm

```diff
@@ -0,0 +1,20 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: This test module launches the installation from
+#          the installation settings page.
+
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use strict;
+use warnings;
+use parent "installbasetest";
```

Comment:
Could you please inherit from `y2_installbase`, as we are doing in another modules, e.g.  https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/f9a8863908aa91811c57dcf218c866d72128b30a/tests/installation/accept_license.pm#L16

Reply:
Then we should update [the documentation](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/ui-framework-documentation.md#1-create-a-test-module-with-the-steps) too, I took the code sample from there.

Reply:
yep, good catch. The documentation was written, probably, when we haven't separate `installbase` from `y2_installbase` yet.